src/LinuxTracing/LibunwindstackUnwinder.cpp

```diff
@@ -101,21 +55,19 @@ const std::array<size_t, unwindstack::X86_64_REG_LAST>
 
 LibunwindstackResult LibunwindstackUnwinderImpl::Unwind(
     pid_t pid, unwindstack::Maps* maps, const std::array<uint64_t, PERF_REG_X86_64_MAX>& perf_regs,
-    const void* stack_dump, uint64_t stack_dump_size, bool offline_memory_only, size_t max_frames) {
+    std::vector<StackSliceView> stack_slices, bool offline_memory_only, size_t max_frames) {
```

Comment:
I think that `stack_slices` should be passed by const reference as it's passed by reference to `CreateWith(out)ProcessMemory`, so moving it into the parameter is not useful.

Reply:
Done.